pkg/cmd/gtctl/cluster/create/config.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package create
+
+import (
+	"fmt"
+	"strings"
+)
+
+const (
+	// Various of support config type
+	configOperator = "operator"
+	configCluster  = "cluster"
+	configEtcd     = "etcd"
+)
+
+type configValues struct {
+	rawConfig []string
+
+	operatorConfig string
+	clusterConfig  string
+	etcdConfig     string
+}
+
+// parseConfig parse raw config values and classify it to different
+// categories of config type by its prefix.
+func (c *configValues) parseConfig() error {
+	var (
+		operatorConfig []string
+		clusterConfig  []string
+		etcdConfig     []string
+	)
+
+	for _, raw := range c.rawConfig {
+		if len(raw) == 0 {
+			return fmt.Errorf("cannot parse empty config values")
+		}
+
+		var configPrefix, configValue string
+		values := strings.Split(raw, ",")
+
+		for _, value := range values {
+			value = strings.Trim(value, " ")
+			config := strings.SplitN(value, ".", 2)
+			configPrefix = config[0]
+			if len(config) == 2 {
+				configValue = config[1]
+			} else {
+				configValue = configPrefix
+			}
```

Comment:
input: `etcd.for=bar`

question: Why does not return failure when the length is not equal 2？

Reply:
cause input `etcd.foo=bar`, the `foo=bar` is for etcd config.

if user input `foo=bar`, apparently it does not specific any config type whatever is going to set. so in that case, i wish gtctl will treat this case as it is for `cluster`, which is the default config type.

so by default, input like `foo=bar` is for `cluster`

Reply:
which is also going to clarify in its usage